package.json

```diff
@@ -18,6 +18,11 @@
           "command": "bootc.image.build",
           "title": "Build Disk Image",
           "when": "ostree.bootable in imageLabelKeys"
+        },
+        {
+          "command": "bootc.vfkit",
```

Comment:
we should also remove the previous part about it being in container page: https://github.com/containers/podman-desktop-extension-bootc/blob/b275f8ff58183aaa2c2de0fea9821f1ea070623d/package.json#L28